augly/image/functional.py

```diff
@@ -1035,12 +1035,13 @@ def overlay_emoji(
     output_path: Optional[str] = None,
     emoji_path: str = utils.EMOJI_PATH,
     opacity: float = 1.0,
-    emoji_size: float = 0.15,
-    x_pos: float = 0.4,
-    y_pos: float = 0.8,
+    emoji_size: Union[float, Tuple[float, float]] = 0.15,
+    x_pos: Union[float, Tuple[float, float]] = 0.4,
+    y_pos: Union[float, Tuple[float, float]] = 0.8,
     metadata: Optional[List[Dict[str, Any]]] = None,
     bboxes: Optional[List[Tuple]] = None,
     bbox_format: Optional[str] = None,
+    seed: int = 42,
```

Comment:
```suggestion
    seed: int = 42,
    metadata: Optional[List[Dict[str, Any]]] = None,
    bboxes: Optional[List[Tuple]] = None,
    bbox_format: Optional[str] = None,
```
nit: augmentation-specific args should go before `metadata`, `bboxes`, & `bbox_format`.